scripts/test_cli/upsert_test.ts

```
@@ -0,0 +1,192 @@
+        )
+        const totalInfo = playlistData.totalInfo.value
+
+        switch (playlistData.ytPlaylistType) {
+            case YTPlaylistTypeDTO.OFFICIAL:
+                const officialPlaylist = new YTOfficialPlaylistTable(
+                    metaDataID,
+                    totalInfo,
+                    playlistData.musicPlatform
+                )
+                await this.client.from('YTOfficialPlaylist').upsert(officialPlaylist).select();
+                await this.client.from("MusicInfoJoin").upsert(musicInfoJoinTable).select();
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)